ModularTegustation/tegu_items/representative/items/lcorp.dm

```diff
@@ -0,0 +1,43 @@
+//See Ncorp file
+/obj/item/limit_increase/lcorp
+	name = "lcorp agent limit breaker"
+	icon_state = "oddity7_firewater"
+
+
+//Officer limit increase
+/obj/item/officerlimit_increase
```

Comment:
This could probably be easily refactored into being based off the N-corp one, just shift the Agent check to the same kind of check this one does.

Reply:
Now it gathers from different areas, so it's not as possible anymore.